README.md

```diff
@@ -48,3 +48,37 @@ using the Octet Encoding Rules (OER).
 Dev-ops is a module for shared build and test tools
 
 [READ MORE](./dev-ops)
+
+### Add Unlimited Strength Encryption Policy Files 
```

Comment:
This looks good to me, too, although I think we should label this as its own section using separator or a label. How about adding a `###` heading called "Buliding the Project"? Otherwise, looks good (though you'll need to submit the PR differently like @earizon indicated).